requirements.txt

```diff
@@ -18,7 +19,7 @@ testtools==1.8.0  # python-subunit
 pbr>=0.11
 testscenarios==0.5.0  # python-subunit
 python-subunit==1.1.0  # lettuce
-requirements/src/lettuce-0.2.21.ccnmtl.tar.gz
+lettuce==0.2.21.ccnmtl
```

Comment:
Okay - I see that you're using the --index-url option now. Thanks for explaining that.